src/gui/MainWindow.cpp

```diff
@@ -275,6 +275,12 @@ void MainWindow::setGtkTouchscreenScrollingForDeviceMapping() {
 }
 
 void MainWindow::setGtkTouchscreenScrollingEnabled(bool enabled) {
+    bool useGtkInertialScroll = control->getSettings()->getGtkTouchInertialScrollingEnabled();
+
+    if (!useGtkInertialScroll) {
```

Comment:
I think it's easier to read if it's on a single line.
```suggestion
    if (!control->getSettings()->getGtkTouchInertialScrollingEnabled()) {
```